velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp

```diff
@@ -1543,6 +1546,42 @@ class ArrowBridgeArrayImportTest : public ArrowBridgeArrayExportTest {
     });
   }
 
+  void testImportStringView() {
```

Comment:
can you also add a test for the schema in `ArrowBridgeSchemaTest.cpp`?

Reply:
Done